CourseTemplate/CourseTemplate.Core/CourseTemplateService.cs

```diff
@@ -17,11 +17,6 @@ public class ViewCourseTemplateDto
     public class CourseTemplateService
     {
         private readonly ICourseTemplateRepository _repository;
-
-        public CourseTemplateService()
-        {
-            _repository = new CourseTemplateRepository();
-        }
```

Comment:
How this impact on migrations applying?

Reply:
This should be deleted in order to launch correctly application in docker. All configuration are done in startup